### PR TITLE
fix: add popular groups onfront page

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/7926
required https://github.com/fossasia/open-event-server/pull/8292

Screenshot from 2021-10-15 08-52-47

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/routes/index.js [hunk 1]
```diff
@@ -203,6 +211,16 @@ export default class IndexRoute extends Route {
...
+      }),
+      promotedGroup: this.store.query('group', {
+        include      : 'user,follower',
+        'page[size]' : 12,
+        filter       : promotedGroupFilter,
+        public       : true,
+        cache        : true
+      }),
+      followedGroups: this.authManager.currentUser.query('followedGroups', {
+        include: 'group,user'
```
[the remark this answers is not in the document]
Comment (pc-beast) [reply app/routes/index.js:223]: Everything else seems to work fine :+1:

**Comment by pc-beast** [comment]
Works now. Test is failing. @maze-runnar

**Comment by maze-runnar** [comment]
I am not getting the actual problem, for some groups it's working, for some it's not.

**Comment by maze-runnar** [comment]
https://github.com/fossasia/open-event-server/issues/6837

**Pull request merged**